toolkit/tools/pkg/imagecustomizerlib/extractpartitions.go

```diff
@@ -114,18 +116,18 @@ func copyBlockDeviceToFile(outDir, devicePath, name string) (filename string, er
 }
 
 // Compress file from .raw to .raw.zst format using zstd.
-func compressWithZstd(partitionRawFilepath string) (partitionFilepath string, err error) {
+func compressWithZstd(partitionRawFilepath string, outputPartitionFilepath string) (partitionFilepath string, err error) {
 	// Using -f to overwrite a file with same name if it exists.
-	err = shell.ExecuteLive(true, "zstd", "-f", "-9", "-T0", partitionRawFilepath)
+	err = shell.ExecuteLive(true, "zstd", "-f", "-9", "-T0", partitionRawFilepath, "-o", outputPartitionFilepath)
 	if err != nil {
 		return "", fmt.Errorf("failed to compress %s with zstd:\n%w", partitionRawFilepath, err)
 	}
 
-	return partitionRawFilepath + ".zst", nil
+	return outputPartitionFilepath, nil
```

Comment:
Do you still need to return `outputPartitionFilepath` anymore?

Reply:
Good point, I do not, I just updated it